com.unity.perception/Runtime/Randomization/Parameters/ICategoricalParameter.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Collections.Generic;
+
+namespace UnityEngine.Perception.Randomization.Parameters
+{
+    /// <summary>
+    /// Exposes the probabilities property of categorical parameters for UI purposes
+    /// </summary>
+    public interface ICategoricalParameter
```

Comment:
Added base categorical parameter class that solves the UI problem I initially used this interface to patch. ICategoricalParameter has since been removed.